services/release/release.go

```diff
@@ -65,7 +65,7 @@ func createTag(ctx context.Context, gitRepo *git.Repository, rel *repo_model.Rel
 
 			commit, err := gitRepo.GetCommit(rel.Target)
 			if err != nil {
-				return false, fmt.Errorf("createTag::GetCommit[%v]: %w", rel.Target, err)
```

Comment:
Why remove the more verbose error message?

Reply:
I removed it as `createTag` and `GetCommit` seemed too implementation-specific to `release_service.CreateRelease()` for it to be included in the API route. I have added an alternative message before printing out the error; let me know if it can be improved!

Reply:
The functions in services will be used by both web side and API.
So if you want to change it for API, it is better to also check the web side.

ps: the alternative message you added is in this PR? L256 in routers/api/v1/repo/release.go?

Reply:
> So if you want to change it for API, it is better to also check the web side.

From reading the web route [code](https://github.com/go-gitea/gitea/blob/e0c27e59968f0ecc5e433617f303520fb765b06a/routers/web/repo/release.go#L483) and observing the web UI, it doesn't seem possible for normal users to pass an invalid target (unless they make create the HTTP request by hand and send it, then they'll get a server error code).

> ps: the alternative message you added is in this PR? L256 in routers/api/v1/repo/release.go?

Yes that is correct. I'm open for better ways of describing the error.